nimble/controller/src/ble_ll_hci.c

```diff
@@ -355,8 +355,11 @@ ble_ll_hci_le_read_bufsize_v2(uint8_t *rspbuf, uint8_t *rsplen)
 {
     struct ble_hci_le_rd_buf_size_v2_rp *rp = (void *) rspbuf;
 
+
+#if MYNEWT_VAL(BLE_LL_ROLE_CENTRAL) || MYNEWT_VAL(BLE_LL_ROLE_PERIPHERAL)
     rp->data_len = htole16(g_ble_ll_data.ll_acl_pkt_size);
     rp->data_packets = g_ble_ll_data.ll_num_acl_pkts;
+#endif
```

Comment:
should be set to 0 if central/perihperal are disabled

Reply:
done